style/img/autonumbering.svg

```diff
@@ -1,4 +1,4 @@
-<svg width="28" height="28" viewBox="0 0 28 28" fill="none" xmlns="http://www.w3.org/2000/svg">
-<rect width="28" height="28" fill="black" fill-opacity="0"/>
-<path d="M4 19H6V19.5H5V20.5H6V21H4V22H7V18H4V19ZM5 10H6V6H4V7H5V10ZM4 13H5.8L4 15.1V16H7V15H5.2L7 12.9V12H4V13ZM9 7V9H23V7H9ZM9 21H23V19H9V21ZM9 15H23V13H9V15Z" fill="#666666"/>
-</svg>
+<svg width="28" height="28" viewBox="0 0 28 28" fill="none" xmlns="http://www.w3.org/2000/svg">

```

Comment:
I strongly advice you to use `LabIcon` registry and store only one version of your icons for all theme.

See https://github.com/jupyterlab/jupyterlab-git/blob/master/src/style/icons.ts

To have icon colors adapt to themes, you will need to define a class to the SVG; see

https://github.com/jupyterlab/jupyterlab-git/blob/91811aa8e127483cc807b26785a8a674397391ac/style/icons/add.svg#L7